chatformatter-core/src/main/java/com/eternalcode/formatter/config/PluginConfig.java

```diff
@@ -25,7 +28,7 @@ public class PluginConfig implements ChatSettings, PlaceholderStack, TemplateRep
     public boolean receiveUpdates = true;
 
 
-    @Description({ " ", "# Chat format for ranks (Vault) Support mini-messages and legacy colors" })
+    @Description({ " ", "# Chat rankFormat for ranks (Vault) Support mini-messages and legacy colors" })
```

Comment:
Can you stay at the `format`? That makes breaking change. This change can be break old user's config.